.github/actions/lint/action.yml

```diff
@@ -15,6 +15,11 @@ runs:
     - run: pip install wheel pre-commit==2.17.0 mypy==0.971 numpy==1.22.4 torch==1.11.0 tensorflow==2.8.3 ray==2.2.0
       shell: bash
       name: install dependencies
+      if: runner.os != 'macOS'
+    - run: pip install wheel pre-commit==2.17.0 mypy==0.971 numpy==1.22.4 torch==1.11.0
+      shell: bash
+      name: install dependencies
+      if: runner.os == 'macOS'
```

Comment:
We already ignore all extra packages in .mypy file. Please, simply remove `numpy==1.22.4 torch==1.11.0 tensorflow==2.8.3 ray==2.2.0`.

Reply:
mypy only ignores missing imports, these are still needed actually there is a PR or two from a while ago.

Reply:
It ignores everything:
diff: https://github.com/mortid0/DeepGNN/commit/921f7226f365e30e593342ee8bcbecc58bff82b2
run: https://github.com/mortid0/DeepGNN/actions/runs/3858029307

Reply:
Thats because I fixed all the bugs from it in a previous pull request already: https://github.com/microsoft/DeepGNN/pull/75/files

Reply:
Do we still have to install numpy, torch, TF etc? If yes, then we should start to enforce it. If no, then remove them.

Reply:
I can do this but that belongs in another PR.

Reply:
Could you elaborate what do you mean by ‘this’?

Reply:
REmoving the mypy rules for numpy, ...